docs/doc_sources/api_reference/dpctl/flags.rst

```diff
@@ -0,0 +1,22 @@
+.. _Flags_class:
+
+``Flags`` class
+===================
+
+.. autoclass:: dpctl.tensor._flags.Flags
+    :members:
+
+    Note that dictionary-like access to some members is permitted:
+
+        "C", "C_CONTIGUOUS":
+            Equivalent to `c_contiguous`
```

Comment:
```suggestion
            Equivalent to ``c_contiguous``
```

Using double quotes would make the text render as code, rather than italicized text.